handbook/company/README.md

```diff
@@ -77,17 +77,17 @@ We work to get results. How we work determines what we get.  Aim to deliver resu
 - **Be realistic.** When you can't take on [more work](http://www.paulgraham.com/hwh.html), clarify your boundaries.  Schedule [time off](https://fleetdm.com/handbook/business-operations#taking-time-off) to recharge.  Practice self-care.
 
 ### 🔵 Objectivity
-To reach our goals, we need to see reality clearly.
+To reach our goals, we need to [see reality clearly](https://en.wikipedia.org/wiki/Intellectual_honesty).
```

Comment:
this link is probably worthwhile.  Could skip if we had to (for example if the merge conflict was just horrible), but otherwise I'd just include it